src/components/sections/Section/Section.scss

```
@@ -0,0 +1,9 @@
+.pages__section {
+  width: 100vw;
+  content-visibility: auto;
+  contain-intrinsic-size: 2px 1024px;
```

I wonder if should we use the 
```suggestion
  contain-intrinsic-size: rem(2px) rem(1024px);
```

along with
```javascript
@import "src/styles/scaffold";
```

Just to keep the pattern if it didn't change the performance, though.